src/components/file.js

```diff
@@ -53,6 +53,10 @@ module.exports = function(app) {
             $scope.files.splice(index, 1);
           };
 
+          $scope.isReadOnly = function() {
+            return $scope.$parent.readOnly
```

Comment:
This is very fragile. What may be better is to add the ```readOnly``` property to the ```formioFilesList``` directive, and then pass along that property using.

```
<formio-files-list .... read-only="readOnly"
```